### PR TITLE
Fixing the way the initial page loads.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

File: src/ReloadTask.ts [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-
-    let gulpConnect = require('gulp-connect');
+  public executeTask(gulp: gulp.Gulp, completeCallback?: (error?: string) => void): void {
+    /* tslint:disable:typedef */
+    const gulpConnect = require('gulp-connect');
+    /* tslint:enable:typedef */
 
     gulp.src('')
-      .pipe(gulpConnect.reload());
+        .pipe(gulpConnect.reload());
```
Comment (dzearing) [inline src/ReloadTask.ts]: nit: unnecessary space

File: src/ServeTask.ts [hunk 2]
```diff
@@ -125,18 +133,14 @@ function logRequestsMiddleware(req: http.IncomingMessage, res: http.ServerRespon
   next();
 }
 
-function enableCorsMiddleware(
-  req: http.IncomingMessage,
-  res: http.ServerResponse,
-  next?: () => any) {
+function enableCorsMiddleware(req: http.IncomingMessage, res: http.ServerResponse, next?: () => void): void {
```
Comment (dzearing) [inline src/ServeTask.ts]: no need to change the params to be all on one line.

File: src/ServeTask.ts [hunk 3]
```diff
@@ -125,18 +133,14 @@ function logRequestsMiddleware(req: http.IncomingMessage, res: http.ServerRespon
...
   res.setHeader('Access-Control-Allow-Origin', '*');
   next();
 }
 
-function setJSONResponseContentTypeMiddleware(
-  req: http.IncomingMessage,
-  res: http.ServerResponse,
-  next?: () => any) {
+function setJSONResponseContentTypeMiddleware(req: http.IncomingMessage,
```
Comment (dzearing) [inline src/ServeTask.ts]: no need to introduce a new spacing convention here.

**Comment by dzearing** [comment]
Approved (with nits)

[Approved with PullApprove](https://pullapprove.com/dzearing/gulp-core-build-serve/pull-request/7/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dzearing/gulp-core-build-serve)

File: src/ServeTask.ts (hunk 2 above)
Comment (iclanton) [inline src/ServeTask.ts]: I'm not a fan of having the arguments on the same line as the code. When skimming the code, it makes it unclear where the argument list ends and where the function body begins. I'm inclined to leave it as-is.

File: src/ServeTask.ts (hunk 3 above)
Comment (iclanton) [inline src/ServeTask.ts]: Same as above.

File: src/ReloadTask.ts (hunk 1 above)
Comment (iclanton) [inline src/ReloadTask.ts]: With this extra space the dots line up :).

**Pull request merged**